### PR TITLE
Russ employee create

### PR DESCRIPTION
**Pull request by Downeastrussell**
Testing and Functionality
When viewing the list of employees
And you click the Create New link
Then a form will be displayed on which the following information can be entered

- First name
- Last name
- checkbox- Is the employee a supervisor
- dropdown-  Select a department

Checklist:
yes --   I have commented my code, particularly in hard-to-understand areas
 no --  I have made corresponding changes to the documentation
 yes --  My changes generate no new errors
 no -- I have added tests that prove my fix is effective or that my feature works
no --  New and existing unit tests pass locally with my changes

### CODE REVIEW

File: BangazonWorkforce/Models/ViewModels/CreateEmployeeViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+{
+    public class CreateEmployeeViewModel
+    {
+        public List<SelectListItem> Departments { get; set; }
+        public Employee Employee { get; set; }
+      
+
+
+
+        public CreateEmployeeViewModel()
```
Comment (ColonelSped) [inline BangazonWorkforce/Models/ViewModels/CreateEmployeeViewModel.cs:19]: What is this code doing? Please add comments to let us know.

**Review by ColonelSped — CHANGES_REQUESTED** [review]

**Comment by ColonelSped** [comment]
Are there some steps on how we should test this code? If so, please add them.

**Comment by nikx1015** [comment]
Everything works 🎉

**Comment by MattRowee** [comment]
Everything works! 🥇

**Review by ColonelSped — APPROVED** [review]

**Comment by ColonelSped** [comment]
👍

**Pull request merged**